src/cbor_encode.c

```diff
@@ -420,7 +420,7 @@ bool multi_encode(uint32_t min_encode,
 		const void *input,
 		uint32_t result_len)
 {
-	if (!PTR_VALUE_IN_RANGE(uint32_t, num_encode, NULL, &max_encode)) {
+	if (!PTR_VALUE_IN_RANGE(uint32_t, num_encode, &min_encode, &max_encode)) {
```

Comment:
I don't see what prompted this in the other changes. Can you elaborate a bit?

Reply:
Nevermind, should have checked the commits specifically rather than the overall diff.